plugins/@grouparoo/postgres/src/lib/table-import/getPropertyValues.ts

```diff
@@ -93,8 +105,21 @@ export const getPropertyValues: GetPropertyValuesMethod = async ({
     query += ` GROUP BY %I`;
     params.push(groupByColumns);
   }
-  if (orderBy.length > 0) {
-    query += ` ORDER BY ${orderBy}`;
+  if (!ranked && orderBys.length > 0) {
+    query += ` ORDER BY ${orderBys.join(", ")}`;
+  }
+
+  if (ranked) {
+    // -- Ranked example
+    // SELECT * FROM
+    //  (SELECT  "user_id" as __pk, "created_at",
+    //              ROW_NUMBER() OVER (PARTITION BY "user_id" ORDER BY "created_at" DESC) AS __rownum
+    //        FROM demo.purchases
+    //        WHERE  "state" = 'successful' AND "user_id" IN ('1','2','3','4')
+    //   ) AS __ranked
+    // WHERE __ranked.__rownum = 1;
```

Comment:
Looks like this should work in Redshift too! https://docs.aws.amazon.com/redshift/latest/dg/r_WF_ROW_NUMBER.html

Reply:
I really want to test against redshift, but there are no mocking libraries I can find, nor will it work for Nock.  It might be time to leave a cluster up and running for test purposes.